### ISSUE TITLE
Add path filtering to scrutinizer config

### ISSUE THREAD

**Issue by sagikazarmark** [portphp/boilerplate#4]
Otherwise test classes are also measured which is bad IMO.

**Comment by ddeboer** [comment]
But tests should be well-written, too. :wink:

**Comment by sagikazarmark** [comment]
Yeah, but they are already part of code testing. So you are checking testing code. Also, I think it is just noise in reports. I am curious about the code, note the tests.
